twine/utils.py

```diff
@@ -57,7 +57,7 @@
 def get_config(path="~/.pypirc"):
     # even if the config file does not exist, set up the parser
     # variable to reduce the number of if/else statements
-    parser = configparser.RawConfigParser()
+    parser = configparser.RawConfigParser(allow_no_value=True)
```

Comment:
Today I learned that `allow_no_value=True` allows for:

```
[pypi]
username
password
```

But with `allow_no_value=False`, `username=` and `password=` are still valid and still result in empty strings.

Reply:
With `allow_no_value=False`, if the config file is of the form:
```
[pypi]
username
password
```
the `username` and `password` key won't be available in the parsed config.
Or am I mistaking?

Reply:
In that case, the parser will raise an Exception, but naked `username` or `password` was not the prescribed syntax; rather `username:` (which I assume is equivalent to `username=`) was.

Reply:
Well, now I remember why I had to add `allow_no_value=True` in order to allow empty value.

What's the difference between adding `:` (or `=`), and using naked keys?
Is it parsed differently?

Reply:
Yes. Adding the separator causes the value to be parsed as the empty string. Not including the separator creates a ParseException unless `allow_no_value=True` in which case the value is `None`.

Reply:
OK. The intent of my PR was to be able to provide a `None` value in the configuration file in order to be anonymous.
Without this PR, you wouldn't be able to provide such value, so the CLI would systematically ask for a login/password (which I didn't want).